src/main/java/io/pravega/connectors/flink/source/reader/PravegaSourceReader.java

```diff
@@ -43,6 +44,7 @@
  *
  * @param <T> The final element type to emit.
```

Comment:
We need to add some more explanation here. Brings the below comments up and add some more on the recovery story with `ExternallyInducedSourceReader` interface.